includes/sqp/1-cf-scoring.md

```diff
@@ -1,63 +1,55 @@
+    | [{{ radarr['cf']['pcm']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#pcm)                         |      :warning: 0 :warning: | {{ radarr['cf']['pcm']['trash_id'] }}             |
     | [{{ radarr['cf']['dts-hd-hra']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#dts-hd-hra)           | :warning: -10000 :warning: | {{ radarr['cf']['dts-hd-hra']['trash_id'] }}      |
-    | [{{ radarr['cf']['ddplus']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#ddplus)                   | :warning: 0 :warning:      | {{ radarr['cf']['ddplus']['trash_id'] }}          |
+    | [{{ radarr['cf']['ddplus']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#ddplus)                   |     :warning: 25 :warning: | {{ radarr['cf']['ddplus']['trash_id'] }}          |
     | [{{ radarr['cf']['dts-es']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#dts-es)                   | :warning: -10000 :warning: | {{ radarr['cf']['dts-es']['trash_id'] }}          |
-    | [{{ radarr['cf']['dts']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#dts)                         | :warning: 0 :warning:      | {{ radarr['cf']['dts']['trash_id'] }}             |
-    | [{{ radarr['cf']['aac']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#aac)                         | :warning: 0 :warning:      | {{ radarr['cf']['aac']['trash_id'] }}             |
-    | [{{ radarr['cf']['dd']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#dd)                           | :warning: 0 :warning:      | {{ radarr['cf']['dd']['trash_id'] }}              |
+    | [{{ radarr['cf']['dts']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#dts)                         |      :warning: 0 :warning: | {{ radarr['cf']['dts']['trash_id'] }}             |
+    | [{{ radarr['cf']['aac']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#aac)                         |      :warning: 0 :warning: | {{ radarr['cf']['aac']['trash_id'] }}             |
+    | [{{ radarr['cf']['dd']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#dd)                           |     :warning: 15 :warning: | {{ radarr['cf']['dd']['trash_id'] }}              |
 
-    !!! warning "Scores marked with a :warning: warning :warning: are different to those used in the main guide"
-
-??? abstract "Movie Versions - [Click to show/hide]"
+    !!! warning "Scores marked with a :warning: warning :warning: are different to those used in the main public guide"
 
-    !!! danger "Adding `IMAX`/`IMAX Enhanced` will replace the BHDStudio release :warning:"
+    !!! danger "The CF with `0` you can choose to add with a score of `0` or just don't add them.<br>The reason why we score them this low is to prevent transcoding as much as possible<br>The reason why `DTS` has a score of `0` is to make sure you don't limit your self to much."
```

Comment:
```suggestion
    !!! danger "The CF with `0` you can choose to add with a score of `0` or just don't add them.<br>The reason why we score them this low is to prevent transcoding as much as possible.<br>The reason why `DTS` has a score of `0` is to make sure, that you don't limit yourself too much."
```